Dockerfile

```diff
@@ -59,11 +59,9 @@ COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
 # non-root
 USER 82
 
-COPY ./composer.json /var/www/html/composer.json
-RUN composer install --no-dev --no-scripts --no-autoloader
+COPY ./composer.json ./composer.lock /var/www/html/
+RUN composer install --no-dev --no-autoloader
 
-COPY . .
```

Comment:
Methodology: 2. Remove `COPY . .` because having access to the php source is only beneficial if the files are defined in autoloaded in composer.json. I've created this issue to re-visit. https://github.com/ministryofjustice/justice-gov-uk/issues/78#issuecomment-2003998333

Reply:
This is quite a confusing subject. Thanks for creating the issue 💯